packaging/nnstreamer.spec

```diff
@@ -362,8 +362,20 @@ Requires:	%{name}-tizen-sensor = %{version}-%{release}
 Tizen Native API wrapper for NNStreamer.
 You can construct a data stream pipeline with neural networks easily.
 
-%post -n capi-nnstreamer -p /sbin/ldconfig
-%postun -n capi-nnstreamer -p /sbin/ldconfig
+# Workaround: Since the rootstrap of Tizen v6.5 is not ready,
+# some application built on v6.0 needs libcapi-nnstreamer.so file.
+# This code will be removed when the rootstrap of Tizen v6.5 is ready.
```

Comment:
It'd be better if you simply add ```Requires: capi-nnstreamer-devel = %{version}-%{release}``` to capi-nnstreamer package as a workaround.

Reply:
If adding `Requires: capi-nnstreamer-devel`, all headers and package config files will be installed on Tizen platform image.

```bash
$ rpm -qlp nnstreamer-devel-1.7.0-0.armv7l.rpm
/usr/include/nnstreamer/nnstreamer_cppplugin_api_filter.hh
/usr/include/nnstreamer/nnstreamer_plugin_api.h
/usr/include/nnstreamer/nnstreamer_plugin_api_converter.h
/usr/include/nnstreamer/nnstreamer_plugin_api_decoder.h
/usr/include/nnstreamer/nnstreamer_plugin_api_filter.h
/usr/include/nnstreamer/nnstreamer_version.h
/usr/include/nnstreamer/tensor_filter_cpp.hh
/usr/include/nnstreamer/tensor_filter_custom.h
/usr/include/nnstreamer/tensor_filter_custom_easy.h
/usr/include/nnstreamer/tensor_typedef.h
/usr/lib/pkgconfig/nnstreamer-cpp.pc
/usr/lib/pkgconfig/nnstreamer.pc
```